### DISCUSSION TITLE
How does a Steam app owner configure their workshop to allow anonymous downloads?

### DISCUSSION THREAD

**Discussion by myk002** [shadoxxhd/steamworkshopdownloader#51] [Q&A] [answered]
The readme says:

List of supported games for anonymous download: https://steamdb.info/sub/17906/apps/

but how does an app get on that list?

**Comment by shadoxxhd** [comment, marked as answer]
A google search led me to these instructions:

1. Log into the steamgames partner site with your publisher account.
2. Go to the page of your app (https://partner.steamgames.com/apps/workshop/[appid]).
3. Scroll down to "Technical tools"
4. Go to the "Edit steamworks" setting
5. Go to the "Workshop" tab, scroll down to the setting "Enable anonymous game servers to download workshop items" and check the checkbox.
6. Hit "Save" on the bottom of the page.
7. Lastly, you probably have to Publish the app

([source](https://community.playstarbound.com/threads/please-enable-public-steam-workshop-downloads.168926/))

No idea if this is still applicable; I don't have a publisher account to test it.

**Reply by myk002** [reply]
This does appear to be the correct setting. Thank you!

![image](https://github.com/shadoxxhd/steamworkshopdownloader/assets/977482/ade02799-a75a-4d9c-8e28-dfba6ce9cfbb)

What confused me about that setting initially is that ISteamUGC is the API from downloading *within the game*, which I didn't think was applicable here.
